{"name": "test_get_articles_with_invalid_query", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 400\n +  where 200 = <Response [200]>.status_code", "trace": "self = <test_articles.TestArticlesAPI object at 0x00000148D0B545D0>, session = <requests.sessions.Session object at 0x00000148CFB5A310>, api_key = '3iElcEAbAjbQuSzc3jRJtU8JVTnA7x5g'\napi_base_url = 'https://api.nytimes.com/svc/'\n\n    def test_get_articles_with_invalid_query(self, session, api_key, api_base_url):\n        params = {\n            \"api-key\": api_key,\n            \"q\": \"\",  # Пустой запрос, должен вернуть ошибку\n        }\n        response = session.get(f\"{api_base_url}{self.endpoint}\", params=params)\n    \n>       assert response.status_code == 400\nE       assert 200 == 400\nE        +  where 200 = <Response [200]>.status_code\n\ntests\\test_articles.py:30: AssertionError"}, "start": 1690385823051, "stop": 1690385823584, "uuid": "3f678907-95bb-4941-817a-5c039607eff9", "historyId": "04569172cdc0f6ac46a00fdba435fad7", "testCaseId": "04569172cdc0f6ac46a00fdba435fad7", "fullName": "tests.test_articles.TestArticlesAPI#test_get_articles_with_invalid_query", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_articles"}, {"name": "subSuite", "value": "TestArticlesAPI"}, {"name": "host", "value": "Chris"}, {"name": "thread", "value": "26012-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_articles"}]}